.circleci/config.yml

```diff
@@ -370,6 +370,11 @@ jobs:
       - checkout
       - restore-gradle-cache
       - init-aws
+      - run:
+          name: update gh
+          command: |
+            apt update
+            apt install gh
```

Comment:
let's avoid updating software as part of CI flow, can we update the CI image instead?

Reply:
Rebuild a new image for this.